omod/src/main/resources/config.xml

```diff
@@ -7,7 +7,7 @@
 	<name>${project.parent.name}</name>
 	<version>${project.parent.version}</version>
 	<package>org.openmrs.module.nmrsmetadata</package>
-	<author>MGIC</author>
```

Comment:
@enumahin This is not necessary, if you need may a change at all..kindly use 'NigeriaMRS Dev Team'.. This is better in my opinion..

Reply:
Lols... And I should also update the same in the clinical module, right?

Reply:
Yes pls..I didn't notice that of the clinical module.